app/views/users/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= render 'layouts/header' %>
+<div class="container pt50">
+    <% flash.each do |message_type, message| %>
+        <div class="alert alert-<%= message_type %>"><%= message %></div>
+    <% end %>
+    <div class="row">
+        <%= @user.name  %>
+        <div class="thumbnail">
+            <%= image_tag @user.image.url, style: "height:300px; width:300px;" %>
```

Comment:
@Masa08  If image is empty, it won't display template properly.
Please add a condition here in case the user did not add an image.